### DISCUSSION TITLE
Registration Discussion

### DISCUSSION THREAD

**Discussion by houssamkhalifeh** [SolarexIT/OpenID-Connect-PHP#6] [Ideas]
This discussion was created with the goal of making it the starting point from which issues will be created and then assigned to any existing *participant* or new joiner to the list of participants by means of taking part in this discussion

**Comment by houssamkhalifeh** [comment]
# Discuss 1
Only members could be mentioned at this point when no comments have been left yet.

**Reply by housssec** [reply]
This is *housssec*. I am replying this comment.

**Reply by houssamkhalifeh** [reply]
@housssec I can now mention you.

**Reply by houssamkhalifeh** [reply]
I will now create an issue, **from this discussion**, and assign it to @housssec
